### PR TITLE
midway merge 

### PR DESCRIPTION
**Pull request by b18phibe**
I wish to merge my changes to the diagram. I have not made enough changes to close any issues but I do have a stable  build whit added features such as the ability to change paper size and to have a page number printed to the bottom right corner of the page.   working on ISSUE #7935 so it will be under this name on the test server.

### CODE REVIEW

File: DuggaSys/diagram.php [hunk 1]
```diff
@@ -256,27 +256,53 @@
...
                             <div class="drop-down-item" tabindex="0">
-                                <span class="drop-down-option" onclick="toggleVirtualA4(event)">Display Virtual A4</span>
-                                <i id="hotkey-displayA4" class="hotKeys">Shift + 4</i>
-                            </div>
-                            <div class="drop-down-item" tabindex="0">
-                                <div id="a4-single-item" class="drop-down-item-disabled">
-                                    <span class="drop-down-option" onclick='togglesingleA4(event);'><img src="../Shared/icons/Arrow_down_right.png">Single A4</span>
+                                <span class="drop-down-option" onclick="toggleVirtualPaper(event)">Display Virtual Paper</span>
+                                <i id="hotkey-displayPaper" class="hotKeys">Shift + 4</i>
+                            </div>
```
Comment (b18simla) [inline DuggaSys/diagram.php:261]: is there a reason for this indentation or is it just a visual bug on github?

Comment (b18simla) [reply DuggaSys/diagram.php:261]: referring to row 262-287

Comment (b18phibe) [reply DuggaSys/diagram.php:261]: it seems like i don't have the right settings on indentations....

Comment (b18phibe) [reply DuggaSys/diagram.php:261]: it's better now

**Pull request merged**